tests/test_enterprise_learner_portal/api/test_views.py

```diff
@@ -190,7 +190,7 @@ def test_patch_missing_params(self, mock_get_overviews, mock_serializer):
         query_params_full = {
             'enterprise_id': str(self.enterprise_customer.uuid),
             'course_id': self.course_run_id,
-            'marked_done': True,
```

Comment:
did not change the test, but verified that both boolean True and string 'true' here work